cclib/method/ddec.py

```diff
@@ -259,7 +269,7 @@ def calculate(self, indices=None, fupdate=0.05):
         self._localizedcharges.append(loc)
         self._stockholdercharges.append(stock)
 
-        # STEP 3
+        # * STEP 3 *
         # Load new proatom densities.
```

Comment:
This is the same description as in step 2 - it's not clear how this is different.

Reply:
I've added some information here about the third step. Thanks.